test/merge-patient-lab-iips-mocha.js

```diff
@@ -1,174 +1,263 @@
 // merge-patient-lab-iips-mocha.js
 
 var chai = require('chai');
+
+var assert = chai.asert;
 var expect = chai.expect;
+
 var chaiAsPromised = require('chai-as-promised');
 chai.should();
 chai.use(chaiAsPromised);
 
+var sinon = require('sinon');
+
 var _ = require('underscore');
-var noflo = require('noflo');
 
 var compFactory = require('../components/merge-patient-lab-iips');
 
 var test = require('./common-test');
 var stubs = require('./common-stubs');
 
-describe('merge-patient-lab-iips', function() {
+describe( 'merge-patient-lab-iips', function() {
```

Comment:
Did not notice that in the guide.  I do it for readability, but recognize it can complicate searching.  I can stop that.